project/dexter-server/public/defect/defectCtrl.js

```diff
@@ -281,56 +278,21 @@ defectApp.controller('DefectCtrl', function ($scope, $http, $sce, $location, $an
         })
     }
 
-    init();
-
-    function checkLogin() {
-        $http.get("/api/v1/accounts/checkLogin", {}).then(function (results) {
-            if (results.data.userId) {
-                $scope.isHideLoginBtnTitle = results.data.userId + "/ logout";
-            }
-            else {
-                $scope.isHideLoginBtnTitle = 'Login';
-            }
-        }, function (results) {
-            $log.error(results);
-        });
-    }
-
-    function initLocalStorage() {
-        window.localStorage['pageSize'] = (window.localStorage['pageSize']) || 500;
-        window.localStorage['currentPage'] = parseInt(window.localStorage['currentPage']) || 1;
-    }
-
-    function getProjectName() {
-        $http.get('/api/v1/projectName', {}).then(function (result) {
-            if (result && result.data) {
-                var html = 'Defect : ' + result.data.projectName;
-                $scope.projectName = result.data.projectName;
-                $('#indexTitle').html(html);
-            }
-        }, function (results) {
-            $log.error(results);
-        });
-    }
-
-    function moveTopOfPage() {
-        $location.hash('bookmark');
-        $anchorScroll();
-    }
-
-    function removeLocalStorageOfResources() {
-        window.localStorage.removeItem('modulePath');
-        window.localStorage.removeItem('fileName');
-        window.localStorage.removeItem('selectTree');
-    }
-
-    var resetAdminUserFlag = function () {
-        $scope.isAdminUser = false;
-        $scope.isCurrentLoginId = false;
+    $scope.loadFileTreeFromDB = function (btnID) {
+        if (btnID !== 'showFileTreeBtn') {
+            return;
+        }
+        if ($scope.isFileTreeHidden === true) {
+            loadTreeItem();
+        }
+        toggleFileTreeAnimation(!$scope.isFileTreeHidden);
+        toggleShowFileTreeBtn(!$scope.isFileTreeHidden);
     };
 
-    moveTopOfPage();
-    resetAdminUserFlag();
+    function loadTreeItem() {
+        $scope.fileTree = [];
+        loadTreeItemFromDB('project', 'project');
+    }
 
     function loadTreeItemFromDB(state, type) {
```

Comment:
for now this code is ok.
Please make loadTreeitemFromDB to service later.
you can refer dexter-monitor project(example)